src/test/documentation_examples/aggregation_data.rs

```diff
@@ -0,0 +1,258 @@
+use bson::doc;
+
+use crate::{error::Result, Database};
+
+pub(crate) async fn populate(db: &Database) -> Result<()> {
+    fn iso_date(text: &str) -> bson::DateTime {
```

Comment:
Can we update this to return a result rather than unwrapping? Also, we should consider waiting til RUST-977 is merged so that we can use the method implemented there rather than going through `chrono`.

Reply:
Updated to use `parse_rfc3339_str`.